app/routes.js

```diff
@@ -344,6 +345,15 @@ const preview_fail = (req, res, dataset_title, datalink, error) => {
   )
 }
 
+const get_line_count = (contentLength, fiveLineString) => {
+   if (contentLength == -1) {
+     return "contentLength"
```

Comment:
nope! sorry should have unassigned you whilst I was still working. Sorry about that